src/app/sign-in/components/two-factor/two-factor.component.scss

```diff
@@ -1,7 +1,20 @@
+.mat-button-font {
+  letter-spacing: 1.25px;
```

Comment:
I believe this is letter-spacing is not required

Reply:
ok I will remove it